<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="oasisSkeleton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>761</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/overlay/OasisProSkeleton.svg)

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/buttons/powerButton.svg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/buttons/upButton.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/buttons/downButton.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tickButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/buttons/tickButton.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="intensityLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow</string>
    </property>
    <property name="text">
     <string>INT</string>
    </property>
   </widget>
   <widget class="QLabel" name="leftEarLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/LeftOff.svg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rightEarLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/RightOff.svg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leftConnectionLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/leftConnectionOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="twentyMinsLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/20minSessionOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="threeHoursLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/3hourSessionOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="customSessionOff">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/CustomSessionOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fortyFiveMinsLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/45minSessionOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sDeltaLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/DeltaSOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="deltaLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/DeltaOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="alphaLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/AlphaOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="betaOneLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/BetaOneOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="betaTwoLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/BetaTwoOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hundredHertzLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/100HzOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="onwLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/OneOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="twoLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/TwoOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="threeLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/ThreeOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fourLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/FourOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fiveLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/FiveOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sixLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/SixOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="eightLabel_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/EightOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="eightLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/SevenOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zeroTwoFiveLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(200,200, 200)</string>
    </property>
    <property name="text">
     <string>0.25</string>
    </property>
   </widget>
   <widget class="QLabel" name="zerooFiveLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(200,200, 200)</string>
    </property>
    <property name="text">
     <string>0.50</string>
    </property>
   </widget>
   <widget class="QLabel" name="zeroSevenFiveLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(200,200, 200)</string>
    </property>
    <property name="text">
     <string>0.75</string>
    </property>
   </widget>
   <widget class="QLabel" name="onwZeroZeroLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(200,200, 200)</string>
    </property>
    <property name="text">
     <string>1.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="pluaTwoLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/PlusTwoOff.svg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="twoZeroZeroLabe">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(200,200, 200)</string>
    </property>
    <property name="text">
     <string>2.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="oneFiveZeroLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(200,200, 200)</string>
    </property>
    <property name="text">
     <string>1.50</string>
    </property>
   </widget>
   <widget class="QLabel" name="pluaOneLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/PlusOneOff.svg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tDCSLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red</string>
    </property>
    <property name="text">
     <string>tDCS</string>
    </property>
   </widget>
   <widget class="QLabel" name="leftConnectionLabel_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/RightConnectionOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="deltaLabel_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/METOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="deltaLabel_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/icons/ThetaOff.svg)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
